view/frontend/web/js/pricing/handlers/ConfigurableProductHandler.js

```
@@ -0,0 +1,189 @@
+            ? this._calculateTaxInclusivePrices(regularPrice, taxConfig)
+            : null;
+
+        // Calculate discount
+        var discount = this._initDiscountRate(regularPrice, finalPrice);
+
+        // Extract rest prices
+        var priceMin = Number(productData.price_min);
+        var priceMax = Number(productData.price_max);
+        var priceMinIncludingTax = this._calculateTaxInclusivePrices(priceMin, taxConfig);
+        var priceMaxIncludingTax = this._calculateTaxInclusivePrices(priceMax, taxConfig);
+        var priceMinRegular = this._extractOriginalPriceFromDiscounted(priceMin, discount);
```

Biome (2.4.6)

[error] This variable is used before its declaration.

(lint/correctness/noInvalidUseBeforeDeclaration)